packages/login_client_flutter/lib/src/flutter_web_secure_credentials_storage.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 LeanCode Sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// ignore: avoid_web_libraries_in_flutter
+import 'dart:html';
+
+import 'package:login_client/login_client.dart';
+
+/// A `flutter_secure_storage` implementation of the [CredentialsStorage].
+class FlutterSecureCredentialsStorage implements CredentialsStorage {
```

Comment:
Actually, LocalStorage is not secure storage. I understand you named this exactly as the iOS/Android class, but I believe this should not be solved that way.

Reply:
If I used another name for this class, I should do this for mobile also. They should have equal names for conditional import